app/src/main/java/in/ac/bits_pilani/goa/ard/utils/Browser.java

```diff
@@ -0,0 +1,278 @@
+     * Sets toolbar color.
+     * @param color The {@link Color}
+     * @return current instance.
+     */
+    public Browser setToolBarColor(@ColorInt final int color) {
+        mBuilder.setToolbarColor(color);
+        return this;
+    }
+
+    /**
+     * Sets the Close button icon for the custom tab.
+     * @param icon The icon {@link VectorDrawable}
+     * @return current instance.
+     */
+    public Browser setCloseButtonIcon(@NonNull final VectorDrawable icon) {
+        final int width = (int) dpToPx(hostActivity, CLOSE_BUTTON_WIDTH);
+        final int height = (int) dpToPx(hostActivity, CLOSE_BUTTON_HEIGHT);
+        mBuilder.setCloseButtonIcon(getBitmapFromVectorDrawable(icon, width, height));
+        return this;
+    }
+
+
+    /**
+     * Sets the start animations.
+     * @param enterResId Resource ID of the "enter" animation for the browser.
+     * @param exitResId Resource ID of the "exit" animation for the application.
+     * @return current instance.
+     */
+    public Browser setStartAnimation(@AnimRes final int enterResId, @AnimRes final int exitResId) {
+        mBuilder.setStartAnimations(hostActivity, enterResId, exitResId);
+        return this;
+    }
+
+    /**
+     * Sets the exit animations.
+     * @param enterResId Resource ID of the "enter" animation for the application.
+     * @param exitResId Resource ID of the "exit" animation for the browser.
+     * @return current instance.
+     */
+    public Browser setExitAnimations(@AnimRes final int enterResId, @AnimRes final int exitResId) {
+        mBuilder.setExitAnimations(hostActivity, enterResId, exitResId);
+        return this;
+    }
+
+    /**
+     * Should url bar hide when user scrolls.
+     * @param value true to hide.
+     * @return current instance.
+     */
+    public Browser setUrlBarHiding(final boolean value) {
+        urlBarHiding = value;
+        return this;
+    }
+
+    /**
+     * Should the title of web-page be shown.
+     * @param value true to show.
+     * @return current instance.
+     */
+    public Browser setShowTitle(final boolean value) {
+        mBuilder.setShowTitle(value);
+        return this;
+    }
+
+    /**
+     * Adds a menu item.
+     * @param label Menu label.
+     * @param pendingIntent Pending intent delivered when the menu item is clicked.
+     * @return current instance.
+     */
+    public Browser addMenuItem(@NonNull final String label, @NonNull final PendingIntent pendingIntent) {
+        mBuilder.addMenuItem(label, pendingIntent);
+        return this;
+    }
+
+    /**
+     * Whether to add default share item in menu.
+     * @param value true to add.
+     * @return current instance.
+     */
+    public Browser setShareInMenu(final boolean value) {
+        shareInMenu = value;
+        return this;
+    }
+
+    /**
+     * Disable action button in toolbar.
+     * @return current instance.
+     */
+    public Browser disableActionButton() {
+        actionButton = null;
```

Comment:
since we are disabliing it by default, can this be directly written in init process instead of creating a method for it and calling it from init?